Sources/Core/State/BufferingState.swift

```diff
@@ -103,10 +103,12 @@ final class BufferingState: NSObject, PlayerState {
         context.player.play()
     }
 
-    func seekCommand(position: Double) {
+    func seekCommand(position: Double, isAccurate: Bool) {
         context.currentItem?.cancelPendingSeeks()
         let time = CMTime(seconds: position, preferredTimescale: context.config.preferredTimescale)
-        context.player.seek(to: time) { [weak self] completed in
+        let toleranceBefore: CMTime = isAccurate ? .zero : .positiveInfinity
```

Comment:
because before and after use the same value, I suggest to simplify and use:
```swift
let tolerance: CMTime = isAccurate ? .zero : .positiveInfinity
```